web/views/tracker_view.ex

```diff
@@ -112,7 +112,7 @@ defmodule Healthlocker.TrackerView do
                   |> Map.get(:hours_slept)
       List.insert_at(list, 0, hours_slept)
     else
-      List.insert_at(list, 0, 0)
+      List.insert_at(list, 0, "null")
```

Comment:
It's used in the js on the front end. We send back a string of 7 comma separated items which the js splits and uses to plot the values on the graph. I can change it to an atom if you think that will help to make things more clear?